### ISSUE TITLE
Landing page - body/about

### ISSUE THREAD

**Issue by tbrlpld** [MozillaFoundation/foundation.mozilla.org#8321]
![image.png](https://images.zenhubusercontent.com/61e1e4f0d1fded7471402d12/f9d862dc-d45d-46ca-938a-2536f6ed17fc)

- Add body/about section to the [research landing page](https://docs.google.com/document/d/19WoZxJvisMsHnXqA4GDX3Te6t9JnqAI2NG5jE9J4LP4/edit#heading=h.etsvp4r9p6tx)
- Title, intro and image
- This is currently the only element we want. Maybe more coming?
- Streamfield with single block maybe? So we are flexible for change?

See also:
- [Research hub models](https://docs.google.com/document/d/19WoZxJvisMsHnXqA4GDX3Te6t9JnqAI2NG5jE9J4LP4/edit#)
- [Designs](https://www.figma.com/file/6BXnlCvPcQrtIaxunvuBYc/Mozilla-Research%3A-Inspiration)

**Comment by beccaklam** [comment]
We are hoping to implement this block for the RCC Playbook as well

- [Design - Figma file](https://www.figma.com/file/dPGD3acNbT75dvEGzNaGUb/Responsible-Computing?node-id=524%3A17550&t=bhKE6OclJCIrq0OH-1)

![image.png](https://images.zenhubusercontent.com/57acd65ce40e5714b16d4215/6c175e44-e2dd-4d4b-93d3-a384c1165927)

**Comment by beccaklam** [comment]
Hi @Gbaldassano we have the desktop, tablet and mobile designs in the [figma file](https://www.figma.com/file/dPGD3acNbT75dvEGzNaGUb/Responsible-Computing?type=design&node-id=524%3A17456&t=JXTDMA0weAUVAAY7-1). Here's a screen shot of what this would look like on [mobile](https://www.figma.com/file/dPGD3acNbT75dvEGzNaGUb/Responsible-Computing?type=design&node-id=524%3A18591&t=JXTDMA0weAUVAAY7-1):
![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/20977376/6d30c74d-c8f2-48b0-84e2-148657960308)

**Comment by danielfmiranda** [comment]
Hi @beccaklam! This has been merged into staging and is now ready to test. To test, visit either the RCC or Research Hub landing page, scroll down to the body section, and create a new "About" Block. 

Note: The image cropping that we have discussed will be implemented in https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10827, at the moment, the div will grow or shrink based on the height of the uploaded image.

Thanks!
